frontend/config/help.yml

```diff
@@ -103,6 +103,15 @@ topics:
   agent_software_notes: '894533774/Agent+Record+Fields'
   agent_software_related_agents: '894664846/Related+Agents+-+Linking+Agent+Record+s+to+an+Agent+Record'
   agent_software_external_documents: '894435542/External+Document+Sub-Record+-+Agent+Records'
+  # Agent Shared
+  agent_sources: -1
```

Comment:
@avatar382 Should code be added in the help controller to skip keys with -1 values?